components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountDisableHandler.java

```diff
@@ -51,6 +56,65 @@ public String getName() {
     }
 
     @Override
+    public String getFriendlyName() {
+        return "Account Disabling";
+    }
+
+    @Override
+    public String getCategory() {
+        return "Login Policies";
+    }
+
+    @Override
+    public String getSubCategory() {
+        return "DEFAULT";
+    }
+
+    @Override
+    public int getOrder() {
+        return 0;
+    }
+
+    @Override
+    public void init(InitConfig initConfig) {
+        super.init(initConfig);
+        AccountServiceDataHolder.getInstance().getBundleContext().registerService
```

Comment:
This is ignored. Since handler should always be registered before connector registers. This is required to initialise the module configs for the connector.